### PR TITLE
v7.2 changelog

### PR DESCRIPTION
**Pull request by amyblais**
To be added: list of contributors.

### CODE REVIEW

File: source/install/self-managed-changelog.md [hunk 1]
```diff
@@ -5,12 +5,154 @@
...
 - [Release v7.0 - Major Release](#release-v7-0-major-release)
 - [Release v6.7 - Feature Release](#release-v6-7-feature-release)
-- [Release v6.6 - Feature Release](#release-v6-6-feature-release)
 - [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
 
+## Release v7.2 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+- **v7.2.0 release day: 2022-08-16**
+
+### Important Upgrade Notes
```
Comment (amyblais) [inline source/install/self-managed-changelog.md:18]: Not sure if there is a way / need to format the below upgrade not better.

Comment (agnivade) [reply source/install/self-managed-changelog.md:18]: > Also, has this been communicated to the customer support team?

Typically we just document the schema changes in upgrade notes. And in cases of major schema changes, we give the support team advance notice. IMO this is not major enough to separately ping the team.

**Review by agnivade — APPROVED** [review]

File: source/install/self-managed-changelog.md [hunk 2]
```diff
@@ -5,12 +5,148 @@
...
+ - Plugin activation errors now show in the plugin management page and marketplace.
+ - Added accessibility to the emoji picker skin tone selector and reversed the order of the skin tone selections in the emoji selector.
+
+#### Administration
+ - Added an "Upgrade" button for Admins on the navigation bar.
+ - Added the ability for Admins to quickly view different paid license options inside the product.
+ - Added the ability to start a trial from the **Invite People** modal.
+ - Added the ability for end users to notify Admins to upgrade their workspace.
+ - Updated the Posts search and get APIs to filter out posts beyond the Cloud plan's limit.
+ - The claim of 10 GB storage per user is no longer shown for grandfathered Cloud plan.
```
Comment (marianunez) [inline source/install/self-managed-changelog.md:81]: This is not relevant for self-hosted - we can remove

File: source/install/self-managed-changelog.md [hunk 3]
```diff
@@ -5,12 +5,148 @@
...
+ - Admins are now able to search for channel IDs via **System Console > User Management > Channels** page.
+ - In the **System Console** left-hand side, paid features icons are now displayed on the menu entries to indicate enterprise features.
+ - Cloud usage rounding for posts and messages is now precise enough to know when a limit has been reached or exceeded.
```
Comment (marianunez) [inline source/install/self-managed-changelog.md:84]: This is not relevant for self-hosted - we can remove

File: source/install/self-managed-changelog.md [hunk 4]
```diff
@@ -5,12 +5,148 @@
...
+### Performance
+ - Removed ``getLastPostPerChannel`` selector for improved performance in channel sorting.
+
+### Bug Fixes
+ - Fixed an issue with pasting a GitHub code snippet in the message box when text is selected.
+ - Fixed an issue where fully typed emojis that contained a capital letter were not correctly displayed.
+ - Fixed an issue where the archived icon did not display correctly in dark themes.
+ - Fixed an issue where password requirements were not enforced when Development Mode was enabled.
+ - Fixed an issue where users were able to attempt to edit the channel header of an archived channel on the right-hand side.
+ - Decreased flakiness of requesting a Cloud trial.
```
Comment (marianunez) [inline source/install/self-managed-changelog.md:99]: This is not relevant for self-hosted - we can remove

**Review by streamer45 — APPROVED** [review]

**Review by marianunez — APPROVED** [review]

**Review by johndavidlugtu — APPROVED** [review]

**Review by rmonga71 — APPROVED** [review]
Thanks @amyblais - re: [MM-46217](https://mattermost.atlassian.net/browse/MM-46217) I've reached out to Suite Users, I'll let you know if there's any further context worth adding to the list of known issues. We believe this is only an issue if compliance export is enabled.

File: source/install/self-managed-changelog.md [hunk 5]
```diff
@@ -5,12 +5,146 @@
...
+ - Added a new static system-level role called Custom Group Manager. This role has permissions to create, edit, and delete custom user groups via User Groups in the Products menu. It can be used to assign individual users this ability when Custom Groups permissions are removed for All Members via the **System Console** (**System Console > Permissions > Edit Scheme > Custom Groups**).
+ - Export file names now contain the ID of the job they were generated by.
+
+### Performance
+ - Removed ``getLastPostPerChannel`` selector for improved performance in channel sorting.
+
+### Bug Fixes
+ - Fixed an issue with pasting a GitHub code snippet in the message box when text is selected.
+ - Fixed an issue where fully typed emojis that contained a capital letter were not correctly displayed.
+ - Fixed an issue where the archived icon did not display correctly in dark themes.
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:95]: The archive icon for channels?

File: source/install/self-managed-changelog.md [hunk 6]
```diff
@@ -5,12 +5,146 @@
...
+ - Fixed an issue where password requirements were not enforced when Development Mode was enabled.
+ - Fixed an issue where users were able to attempt to edit the channel header of an archived channel on the right-hand side.
+ - Fixed an issue where the “Your Trial Ended” banner hid the product switcher menu.
+ - Fixed an issue where the custom status date format was not set to ``yyyy-MM-dd``.
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:99]: Question: Is MM intended when the rest of the date format is in lowercase (yyyy dd)?

File: source/install/self-managed-changelog.md [hunk 7]
```diff
@@ -5,12 +5,146 @@
...
+ - Updated permissions of the ``api/v4/posts/{post_id:[A-Za-z0-9]+}/thread`` endpoint. If [compliance monitoring](https://docs.mattermost.com/comply/compliance-monitoring.html) is enabled, a user can on longer view threads in a public channel they are not a member of.
+
+### Go Version
+ - v7.2 is built with Go ``v1.18.1``.
+
+### Open Source Components
+ - Added ``@types/color-hash``, ``color-contrast-checker``, ``color-hash``, and ``webpack`` to https://github.com/mattermost/mattermost-webapp.
+
+### Known Issues
+ - Forwarding messages: hitting the "enter" key on an auto-complete item in the comment box sends the forward message [MM-46142](https://mattermost.atlassian.net/browse/MM-46142).
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:132]:
```suggestion
 - Forwarding messages: pressing Enter key on an auto-complete item in the comment box sends the forward message [MM-46142](https://mattermost.atlassian.net/browse/MM-46142).
```

**Review by gabrieljackson — APPROVED** [review]

**Review by johndavidlugtu — APPROVED** [review]

**Pull request merged**